static/src/javascripts/projects/commercial/modules/cmp/consent-tracker.js

```diff
@@ -30,7 +27,7 @@ const buildPayload = (
 ): ConsentPayload => ({
     pv: pageViewId,
     cs: consent.consentData,
-    cc: getAdConsentState(thirdPartyTrackingAdConsent),
+    cc: consentState('advertisement'),
```

Comment:
We probably need to understand what this does and how passing a different payload to it will affect things upstream.

Reply:
I've since noticed that `consentState('advertisement')` is a boolean as well so this might be fine! I think that raises other questions here and elsewhere though. Is the behaviour consistent with the new meaning of advertising consent here? How are we mapping the IAB purposes, and what if someone wants to be able to see specifics for the IAB purposes (rather than a boolean reduction)?

Reply:
For now and until we start using the new CMP UI `consentState('advertisement')` can be equal to `getAdConsentState(thirdPartyTrackingAdConsent)`. 

> How are we mapping the IAB purposes?

We're still waiting on direction as to what `consentState('advertisement')` will return after the CMP UI is in use, it'll be a value based on user's responses to 5 IAB purposes, eg. It could be we need all 5 IAB purposes to be `true` for it to return `true`, or maybe we only need a subset to be `true`.

> what if someone wants to be able to see specifics for the IAB purposes (rather than a boolean reduction)?

If user's of the IAB purposes need more granular specifics about the IAB purposes these will be available via the IAB string (and the 3 supported methods of accessing it).

I'm unsure whether third parties that honour the higher level `consentState('advertisement')` setting are part of the IAB framework? Maybe @ripecosta can clarify this.

Reply:
The third party tags that are behind the `consentState('advertisement')` (whether directly or through the callback) are either not part of the IAB framework (most cases) or are part of the IAB but have first party code that needs to be checked against consent (such reading the Krux targeting cookie).

Reply:
I see :-)

We're actually integrating the existing CMP with this new framework as well. While that does make sense I do wonder whether that is another separate thing in the same PR!

I agree with your points about the IAB string, there's a similar point on another of my comments lower down.